index.js

```diff
@@ -1,10 +1,187 @@
-const http = require('http')
 
-const app = http.createServer((request, response) => {
-    response.writeHead(200, { 'Content-Type': 'text/plain' })
-    response.end('Hello World')
+require('dotenv').config()
+const express = require('express')
+const app = express()
+const bodyParser = require('body-parser')
+const morgan = require('morgan')
+const cors = require('cors')
+const Person = require('./models/person')
+
+
+// tad kā? express.json() un/vai bodyParser? atkāpties un noskaidrot
+app.use(express.json())
+app.use(bodyParser.json())
+
+
+app.use(cors())
+app.use(express.static('build'))
+
+
+// eslint-disable-next-line no-unused-vars
+morgan.token('body', (req, res) => JSON.stringify(req.body))
```

Comment:
`// eslint-disable` rūļus likt kodā nevajadzētu, tas ir tā saucamais ["code smell"](https://en.wikipedia.org/wiki/Code_smell#mw-content-text:~:text=code%20smell%20is%20any%20characteristic%20in,that%20possibly%20indicates%20a%20deeper%20problem).

ESLint būtība ir tieši par to, norādīt aizmāršības kodā, kuras tu uzreiz vari salabot. Labot viņas ar disable rūli ir kind of counterintuitive.

[Why did you even get the no-unused-vars in the first place?](https://eslint.org/docs/rules/no-unused-vars#disallow-unused-variables-no-unused-vars:~:text=Variables%20that%20are%20declared%20and%20not,can%20lead%20to%20confusion%20by%20readers.)

Reply:
Eh, šeit es pildīju pēc piemēra. Tur bija salikts gan šis, gan eqeqeq

Reply:
Bet par šo man bija jautājums jau sen - te visu laiku pieprasījumos ir neizmantotie - request, response  ... error utml., kā kuru reizi. 
Un tos neizmanto. Vai tie būtu dzēšami ārā? Es nedzēšu, jo negribu izjaukt ritmu un kārtību, ja nu tāds tur ir.
Bet, jā, šo es gribēju noskaidrot jau sen - kas jādara? Atstāt vai dzēst?

Reply:
Neizmantotais tiek vienmēr dzēsts ārā.

Mums projektos ESLint un Prettier dzied kopā un saglabājot failu ESLint izpilda autofix, where applicable un Prettier pēc tam saformatē.

Developerim tas ir pilnībā caurspīdīgs un nejūtams process. Ir reta reize, kad paliek kaut kas pāri un tad to explicitly ar roku nohandlo. Unused variables ir 100% izķerts gadījums un tu nekad ar šo nesaskartos.

Reply:
Piemērā... tu domā FSO piemērā? Ja tā, tad kārtējais kauns viņiem. Ja citur, tad tas visdrīzāk izriet no tā, ka FSO neizstāsta līdz galam ESLint būtību. Pasaka, ka vajag, jūs kā studenti uzliekat, neizprotat visas nianses un maujat tālāk.

Tādēļ jau tev ir tas upper hand, ka mums ir PR process tavam FSO, kur var in-depth pārskatīt tavu gaitu.